pyaedt/modeler/cad/Primitives.py

```diff
@@ -1480,26 +1480,40 @@ def refresh_all_ids(self):
         return len(self.objects)
 
     @pyaedt_function_handler()
-    def get_objects_by_material(self, materialname):
-        """Retrieve a list of the IDs for objects of a specified material.
+    def get_objects_by_material(self, materialname=None):
+        """Retrieve a list of objects of a specified material
+        or retrieve a list of objects classified by material.
```

Comment:
If this is too long (Alex initially insisted on one-line descriptions), it could be changed to "Retrieve a list of objects either of a specified material or classified by material."

Reply:
Hi @PipKat , thanks for your feedback and suggestions!
We are actually tryinig to be more exhaustive in the documentation since it's one of the few concerns we are receiving from customers.